components/filter/filter.js

```diff
@@ -573,13 +573,12 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 
 	_dispatchChangeEvent(dimension, change) {
 		this._setDimensionChangeEvent(dimension, change, false);
+		clearTimeout(this._changeEventTimeout);
```

Comment:
So just to make sure I'm understanding the change correctly - instead of simply batching the events and running every 200ms, we now wait until there's been 300ms of inactivity before we send them. Is this necessary for the changes in https://github.com/BrightspaceHypermediaComponents/common/pull/52, or just helpful to reduce the HM calls over there?

Overall it seems like a good idea that even non-HM filter uses could benefit from, but want to make sure I understand the context.

Reply:
Yes, now we will fire event after 300ms of inactivity. I changed to 300s as normally when selecting too fast, it seemed to be good enough time to select multiple filters, that will reduce the triggered events. It would reduce the the number of fired events for both HM and non-HM filters